Jantimizer/Histograms/Managers/PostgresEquiDepthHistogramManager.cs

```diff
@@ -85,15 +86,32 @@ private async Task<DataTable> GetAttributenamesForTable(string tableName)
         private async Task AddHistogramForAttribute(DataRow row, string tableName)
         {
             string attributeName = $"{row["column_name"]}".ToLower();
-            var attributeValues = await DbConnector.CallQuery($"SELECT {attributeName} FROM {tableName}");
-            if (attributeValues.Tables.Count > 0)
-            {
-                IHistogram newHistogram = new HistogramEquiDepth(tableName, attributeName, Depth);
-                newHistogram.GenerateHistogram(attributeValues.Tables[0], attributeName);
-                Histograms.Add(newHistogram);
-            }
+            DataSet sortedGroupsDs = await DbConnector.CallQuery(@$"
+                SELECT
+                    {attributeName} AS val,
+                    COUNT({attributeName})
+                FROM {tableName} WHERE
+                    {attributeName} IS NOT NULL
+                GROUP BY {attributeName}
+                ORDER BY {attributeName} ASC
+            ");
+
+            List<ValueCount> sortedGroups = GetValueCounts(sortedGroupsDs, "val", "COUNT").ToList();
+
+            IHistogram newHistogram = new HistogramEquiDepth(tableName, attributeName, Depth);
+            newHistogram.GenerateHistogram(sortedGroups);
+            Histograms.Add(newHistogram);
         }
```

Comment:
Why was this only added to the Postgres manager and not the MySQL manager?

Reply:
I forgot it at the time, but it already works with MySQL continuing in its own way,
so I vote for letting that be part of #108